readthedocsext/theme/templates/projects/import_config.html

```diff
@@ -12,43 +12,33 @@
     {% blocktrans trimmed %}
       A <code>.readthedocs.yaml</code> configuration file is required at the root of your repository in order to build your documentation.
     {% endblocktrans %}
-
-    <a href="https://docs.readthedocs.io/page/config-file/index.html" target="_blank">
-      {% trans "Learn how to add a configuration file to your project." %}
-    </a>
   </div>
 
-  <div class="ui basic fitted right aligned segment">
-    <span>{% trans "Example configuration for:" %}</span>
-    <div class="ui inline dropdown" data-bind="semanticui: { dropdown: {}}">
-      <input type="hidden" name="tool" value="sphinx">
-      <span class="text">Sphinx</span>
-      <i class="dropdown icon"></i>
-      <div class="menu">
-        <div class="item" data-value="sphinx">Sphinx</div>
-
-        {% comment %}
-          Adding a second option here will require a bit of JS or FUI:
-          https://github.com/readthedocs/ext-theme/issues/184
-        {% endcomment %}
-
-        {# The `actionable` class here prevents the select from selecting the text #}
-        <a class="actionable item" href="https://docs.readthedocs.io/page/config-file/index.html#getting-started-with-a-template" target="_blank">
-          {% trans "See more examples" %}
-        </a>
-      </div>
-    </div>
-  </div>
+  <p>
 
-  <div class="ui mini padded inverted scrolling segment">
-    <div class="ui top attached label">
-      <code>.readthedocs.yaml</code>
-    </div>
-    <a class="ui top right attached icon label" href="#" data-clipboard-target="#project-create-sample-sphinx">
-      <i class="fas fa-copy icon"></i>
-    </a>
-    <code class="highlight">
-      <pre id="project-create-sample-sphinx">
+    Here you have some simple examples for the most common documentation tools.
+    If you are using a different tool, you can read our documentation to
+    <a href="https://docs.readthedocs.io/page/config-file/index.html" target="_blank">learn how to write your own</a>.
+  </p>
+
+  <div class="ui top attached tabular menu">
+    <div class="active item" data-tab="sphinx">Sphinx</div>
+    <div class="item" data-tab="mkdocs">MkDocs</div>
+    <div class="item" data-tab="pelican">Pelican</div>
+    <div class="item" data-tab="docusaurus">Docusaurus</div>
+    <div class="item" data-tab="jekyll">Jekyll</div>
+    <div class="item" data-tab="others">Others</div>
```

Comment:
This needs localization:

```suggestion
    <div class="item" data-tab="others">{% trans "Others" %}</div>
```